src/kvstore/raftex/RaftPart.cpp

```diff
@@ -962,7 +966,7 @@ bool RaftPart::needToSendHeartbeat() {
     std::lock_guard<std::mutex> g(raftLock_);
     return status_ == Status::RUNNING &&
            role_ == Role::LEADER &&
-           lastMsgSentDur_.elapsedInSec() >= FLAGS_raft_heartbeat_interval_secs * 2 / 5;
+           lastMsgSentDur_.elapsedInMSec() >= FLAGS_raft_heartbeat_interval_secs * 1000 * 2 / 5;
```

Comment:
Could using `FLAGS_raft_heartbeat_interval_ms` ?

Reply:
That would be better, I just want to reserve the name `raft_heartbeat_interval_secs`.